packaging/conda/meta.yaml

```diff
@@ -36,16 +36,57 @@ requirements:
   run:
     - {{ mpi }}
 
+# requirements:
+#   run:
+#     - khiops-core
+#     - kni
+#     - kni-transfer
+
 outputs:
   - name: khiops-core
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - {{ compiler('cxx') }}
+      host:
+        - {{ mpi }}
+      run:
+        - {{ mpi }}
+    files:
+      - bin/MODL*
     test:
       commands:
         - MODL -v
         - MODL_Coclustering -v
+  - name: kni
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    files: 
+      - bin/libKhiopsNativeInterface* # [linux or osx]
+      - bin/KhiopsNativeInterface.dll # [win]
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - {{ compiler('cxx') }}
+  - name: kni-transfer
+    version: {{ os.environ.get('KHIOPS_VERSION') }}
+    requirements:
+      build:
+        - cmake
+        - ninja
+        - {{ compiler('cxx') }}
+      run:
+        - kni 
+    files: 
+      - bin/KNITransfer*
```

Comment:
Why the glob here? Is there for the Windows `.exe` extension? If so, then why not put both, with `[linux or osx]` vs. `[win]`?